com.civicrm.hrjobroles/js/src/job-roles/controllers/job-roles.controller.js

```diff
@@ -477,12 +477,13 @@ define([
     }
 
     /**
-     * Trigger opening of editor window for editing provided option type
+     * Open editor for option editing
```

Comment:
The descriptions should say what function does, not what you could do with it. So `Opens` instead of `Open`.

NO: I can "Open editor for option editing"
YES: It (function) "Opens editor for option editing"

Reply:
Alright noted.